apps/point-of-sale/src/components/Cart/CartComponent.vue

```diff
@@ -68,7 +68,7 @@ const transactions = ref<BaseTransactionResponse[]>([]);
 
 const getUserRecentTransactions = () => {
   transactions.value = [];
-  if (cartStore.getBuyer) {
+  if (cartStore.getBuyer && cartStore.getBuyer.id === authStore.getUser?.id) {
```

Comment:
Do the recent transaction for a user need to be displayed in borrelmode?

Reply:
Yes, this is used when a BAC'er forgets if they entered a transaction or not.